CHANGES.rst

```diff
@@ -157,6 +158,7 @@ version number.  Breaking changes may occur at any point.
     with ``go vet`` [GH-765] [GH-897]
   - Add ``flycheck-ghc-stack-use-nix`` to enable Nix support for Stack GHC
     [GH-913]
+  - Removal of the Jade checker [GH-951] [GH-1084]
```

Comment:
That's the wrong version to add the note to.  Please rebase onto `master` and add the change notes to version 30 which is currently in development.

Furthermore, please keep the grammar and phrasing consistent to the other entries, i.e. not "Removal of", but just "remove".